File: project-docker/python-scripts/result/PythonResult.py
import json

result = {
    "isLogin": False,
    "isGetCourseId": False,
    "user":{},
    "courses": [],
    "questionCount":0,
    "subsectionTdList":[],
    "finishContent":"",
    "finishCount":0
}


def getResultFalseLogin():
    print(json.dumps(result))
def getResultFalseGetCourseId():
    print(json.dumps(result))

def getResultTrueLogin(userId, userPassword, userName):
    result["user"]["userId"] = userId
    result["user"]["userPassword"] = userPassword
    result["user"]["userName"] = userName
    result["isLogin"] = True
    print(json.dumps(result))

File: javaServer_AnswerToQuestion/main/answer-question-service/src/main/java/com/smileShark/answerQuestion/common/PythonResult.java
package com.smileShark.answerQuestion.common;

import com.smileShark.answerQuestion.entity.Course;
import com.smileShark.answerQuestion.entity.User;
import lombok.Data;
import org.springframework.stereotype.Component;

import java.util.List;

@Data
@Component
// 返回的python结果接收
public class PythonResult {
    private Boolean isLogin;
    private User user;
    private Boolean isGetCourseId;
    private List<Course> courses;
    private Integer questionCount;
    private List<String> subsectionTdList;
    private String questionData;
    private String finishContent;
    private Integer finishCount;
}
